get-started/09_image_best.md

```diff
@@ -197,30 +182,26 @@ a change to the `package.json`. Make sense?
 5. Build the Docker image now using `docker build -t getting-started .` again. This time, your output should look a little different.
 
     ```plaintext
-    Sending build context to Docker daemon  219.1kB
-    Step 1/6 : FROM node:12-alpine
-    ---> b0dc3a5e5e9e
-    Step 2/6 : WORKDIR /app
-    ---> Using cache
-    ---> 9577ae713121
-    Step 3/6 : COPY package.json yarn.lock ./
-    ---> Using cache
-    ---> bd5306f49fc8
-    Step 4/6 : RUN yarn install --production
-    ---> Using cache
-    ---> 4e68fbc2d704
-    Step 5/6 : COPY . .
-    ---> cccde25a3d9a
-    Step 6/6 : CMD ["node", "src/index.js"]
-    ---> Running in 2be75662c150
-    Removing intermediate container 2be75662c150
-    ---> 458e5c6f080c
-    Successfully built 458e5c6f080c
-    Successfully tagged getting-started:latest
+    [+] Building 1.2s (10/10) FINISHED
+    => [internal] load build definition from Dockerfile
+    => => transferring dockerfile: 37B
+    => [internal] load .dockerignore
+    => => transferring context: 2B
+    => [internal] load metadata for docker.io/library/node:18-alpine
+    => [internal] load build context
+    => => transferring context: 450.43kB
+    => [1/5] FROM docker.io/library/node:18-alpine
+    => CACHED [2/5] WORKDIR /app
+    => CACHED [3/5] COPY package.json yarn.lock ./
+    => CACHED [4/5] RUN yarn install --production
+    => [5/5] COPY . .
+    => exporting to image
+    => => exporting layers
+    => => writing image     sha256:91790c87bcb096a83c2bd4eb512bc8b134c757cda0bdee4038187f98148e2eda
+    => => naming to docker.io/library/getting-started
     ```
 
-    First off, you should notice that the build was MUCH faster! And, you'll see that steps 1-4 all have
-    `Using cache`. So, hooray! We're using the build cache. Pushing and pulling this image and updates to it
+    First off, you should notice that the build was MUCH faster! And, you'll see that several steps are using previously cached layers. So, hooray! We're using the build cache. Pushing and pulling this image and updates to it
```

Comment:
MUCH > much

Reply:
Thanks, @jerae-duffin. I'll create a separate PR to update the styles in this topic.